yamls/kind.yaml.j2

```diff
@@ -78,5 +78,8 @@ nodes:
     image: kindest/node:{{ k8s_version }}
     labels:
       kube-ovn/role: master
+  {%- else %}
+  - role: worker
+    image: kindest/node:{{ k8s_version }}
```

Comment:
Add another node only for ovn-ic e2e tests.

Reply:
fixed